src/managers/RoleManager.js

```diff
@@ -273,7 +273,7 @@ class RoleManager extends CachedManager {
     const resolvedRole1 = this.resolve(role1);
     const resolvedRole2 = this.resolve(role2);
     if (!resolvedRole1 || !resolvedRole2) throw new TypeError('INVALID_TYPE', 'role', 'Role nor a Snowflake');
-    if (resolvedRole1.position === resolvedRole2.position) return resolvedRole2.id - resolvedRole1.id;
+    if (resolvedRole1.position === resolvedRole2.position) return BigInt(resolvedRole2.id) - BigInt(resolvedRole1.id);
```

Comment:
wouldn't this change the return type to `number|bigint`?

Reply:
Yep, thank you for pointing that out. Fixed with 35922f5a6ba4286e93948daa9b1c580827156553.